src/views/quote_view.js

```diff
@@ -0,0 +1,76 @@
+import Backbone from 'backbone'
+import Quote from '../models/quote'
+
+const QuoteView = Backbone.View.extend({
+  // initialize is called when a new instance of QuoteView is created
+  // we will pass an underscore template in the params
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus;
+    // this.render will be called when an attribute of a model is changed (aka when the Quote buy and sell methods run)
+    this.listenTo(this.model, 'change', this.render);
+    // listen for a custom event from OrderView to buy the stock if the price is right
+    this.listenTo(this.bus, `buy${this.model.get('symbol')}`, this.buyStock)
+    // litsten for a custom event from OrderView to sell the stock if the price is right
+    this.listenTo(this.bus, `sell${this.model.get('symbol')}`, this.sellStock)
+  }, // initialize
+  // in render we are compiling a template and then we are setting the inside of $el to be the html from compiledTemplate
+  // $el is the jQuery selection of the element that encompasses our view
+  render() {
+    console.log('in the QuoteView render');
+    const compiledTemplate =
+    this.template(this.model.toJSON());
+    this.$el.html(compiledTemplate);
+
+    // this event will be listened to in OrderListView
+    // the Order will listen to this event check the current price of the Quote (passed from this function to OrderListView via the bus) againts the targetPrice for the Order to determine if it is time to buy or sell the order yet.
+    let priceOfQuote = this.model.get('price');
+    let nameOfQuote = this.model.get('symbol')
+    this.bus.trigger(nameOfQuote, priceOfQuote)
+
```

Comment:
This view can trigger two types of event on the bus: `boughtOrSold`, where the data is info about the trade, and one named after the symbol, where the data is the current price. They're also triggered in sequence - changing the stock price from `buyStock` or `sellStock` will trigger an `update` and a `render`, resulting in the symbol-name event.

It ought to be possible to consolidate this and simplify your events a bit.